metric_learn/base_metric.py

```diff
@@ -641,4 +641,4 @@ def score(self, quadruplets):
     score : float
       The quadruplets score.
     """
-    return - np.mean(self.predict(quadruplets))
+    return np.mean((self.predict(quadruplets) + 1) / 2)
```

Comment:
This should be equivalent to `pred.mean() / 2  + 0.5`, which is more efficient to compute.

I'd also add a comment explaining that the prediction is a vector of either -1 or 1 values, and that we're rescaling to [0, 1].

Reply:
I agree, done